src/ResourceManager/Compute/Commands.Compute/Generated/ContainerService/Config/RemoveAzureRmContainerServiceAgentPoolProfileCommand.cs

```diff
@@ -42,6 +42,7 @@ public class RemoveAzureRmContainerServiceAgentPoolProfileCommand : Microsoft.Az
         [Parameter(
             Mandatory = true,
             Position = 1,
+            ParameterSetName = "NameParameterSet",
```

Comment:
If this is concerned, I will update our generator and remove parameter set when there is only one way to remove.